civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -4675,6 +4676,52 @@ function civihr_employee_portal_form_user_profile_form_alter(&$form, &$form_stat
 }
 
 /**
+ * Implements hook_file_presave().
+ *
+ * @param \stdClass $file
+ */
+function civihr_employee_portal_file_presave($file) {
+  $resizeSources = [
+    'submitted_civicrm_1_contact_1_contact_image_URL'
+  ];
+
+  if (in_array($file->source, $resizeSources)) {
+    ImageHelper::resizeForProfile($file->uri);
+  }
+}
+
+/**
+ * @param string $formName
+ * @param CRM_Contact_Form_Contact $form
+ */
+function civihr_employee_portal_civicrm_postProcess($formName, &$form) {
+  if ($formName !== 'CRM_Contact_Form_Contact') {
```

Comment:
I think this would be more clean : 
```php
switch($formName){
case 'CRM_Contact_Form_Contact':
   _civihr_employee_portal_resize_civicrm_contact_image($formName, &$form);
break;
}

```

where the rest of code below go into _civihr_employee_portal_resize_civicrm_contact_image() ..

Reply:
I don't like the ever expanding size of the module file, so I moved this into a static method in another class.